nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -281,46 +287,74 @@ public void renameTable(TableIdentifier from, TableIdentifier toOriginal) {
     // behavior. So better be safe than sorry.
   }
 
-  /**
-   * creating namespaces in nessie is implicit, therefore this is a no-op. Metadata is ignored.
-   *
-   * @param namespace a multi-part namespace
-   * @param metadata  a string Map of properties for the given namespace
-   */
   @Override
   public void createNamespace(Namespace namespace, Map<String, String> metadata) {
+    try {
+      api.createNamespace()
+          .refName(reference.getName())
```

Comment:
Should be `.reference(reference)` (for all other occurences below as well)

Reply:
done